app/safe/internal/bootstrap/bootstrap.go

```diff
@@ -188,7 +188,7 @@ func CreateCryptoKey(id *string, updatedSecret chan<- bool) {
 
 	log.InfoLn(id, "Generated public key, private key, and aes seed")
 
-	if err = persistKeys(privateKey, publicKey, aesSeed); err != nil {
+	if err = PersistKeys(privateKey, publicKey, aesSeed); err != nil {
```

Comment:
Can we add a comment here?

Something like:

```go
// Save the key to VSecM Safe's memory, and also save them to 
// VSecM Safe's root key Kubernetes Secret.
// If the root key input mode is not manual, VSecM Safe will
// use a trusted backing store to retrieve the key in case of
// Pod crash or eviction. As of Mar,17, 2024 the only trusted
// backing store is the VSecM root key Kubernetes Secret; however
// this will change in the future.
if err = PersistKeys(privateKey, publicKey, aesSeed); err != nil {
```

Reply:
Reasoning: I always forget why we didn’t do a "IsManualRootKeySavedAsK8sSecret()" check here; so it’s reminder to me, and also anyone who thinks the same too :) .